queries/risk_free_batches.sql

```diff
@@ -0,0 +1,42 @@
+with interactions as (select selector,
+                             target,
+                             case
+                                 when selector in (
+                                                   '\x095ea7b3', -- approve
+                                                   '\x2e1a7d4d', -- withdraw
```

Comment:
How likely are collusions that the same selector means different things on different contracts? Could you drop a comment about it?

Reply:
This query shows a complete list of all different selectors used on our protocol and so far there are no collisions. We can monitor this for new risk free events:

https://dune.com/queries/1434731

I will comment about this in the code.